samply/src/windows/mod.rs

```diff
@@ -490,11 +490,17 @@ impl ProfileContext {
         // Virtualised ARM64 Windows crashes out on PROFILE tracing, and that's what I'm developing
         // on, so these are hacky args to get me a useful profile that I can work with.
         xperf.arg("-on");
-        //xperf.arg("PROC_THREAD+LOADER+PROFILE");
-        xperf.arg("PROC_THREAD+LOADER+CSWITCH+SYSCALL+VIRT_ALLOC+OB_HANDLE");
+        if self.arch != "aarch64" {
+            xperf.arg("PROC_THREAD+LOADER+PROFILE");
+        } else {
+            xperf.arg("PROC_THREAD+LOADER+CSWITCH+SYSCALL+VIRT_ALLOC+OB_HANDLE");
+        }
         xperf.arg("-stackwalk");
-        //xperf.arg("profile");
-        xperf.arg("VirtualAlloc+VirtualFree+HandleCreate+HandleClose");
+        if self.arch != "aarch64" {
+            xperf.arg("profile");
+        } else {
+            xperf.arg("VirtualAlloc+VirtualFree+HandleCreate+HandleClose");
```

Comment:
I don't really understand this. Doesn't this mean that we don't get any time-based stacks on aarch64?

Reply:
No, this is currently just while I'm hacking on this -- for whatever reason, time based stacks don't work on aarch64 VMs. They work fine on physical aarch64 hardware, but I'm on a MBP with Parallels, so.. :|

Reply:
ah, makes sense